server/controllers/users/getUsers/index.js

```diff
@@ -0,0 +1,12 @@
+const { getAllUsers } = require('../../../database/queries');
```

Comment:
Rename this module to `getUsersQuery` or rename this module name to `getAllUsers`